pages/changelog.md

```diff
@@ -1,53 +1,55 @@
-### [Website]- Add ability to change which solutions are published:
+# Changelog
+
+## [Website]- Add ability to change which solutions are published:
 
 Previously, you could not change which solutions are published. We have now enabled this along with the ability to choose which appear on the profile and which allow public comments. [#4163](https://github.com/exercism/exercism/issues/4163)
 
-### [Website] Add communal mentoring notes
+## [Website] Add communal mentoring notes
 We have exposed the communal mentoring notes from GitHub to be visible directly within the UI.
 
-### [Teams] Launch the site
+## [Teams] Launch the site
 
 We have built a dedicated site for teams. It allows groups of people to complete exercises and mentor each other in a private space. We envision it being used by bootcamps, groups of friends, and colleagues at work.
 
-### [Website] Delete my account
+## [Website] Delete my account
 
 We needed to give people the ability to delete their account without emailing Jeremy.
 
-### [Website] Reset CLI token
+## [Website] Reset CLI token
 
 We wanted to let people reset their CLI tokens in case they are leaked. [#4150](https://github.com/exercism/exercism/issues/4150)
 
-### [Website] Improve experience of "Legacy" or "Practice" solutions in Mentored Mode
+## [Website] Improve experience of "Legacy" or "Practice" solutions in Mentored Mode
 
 Have separate restricted queues for core and side exercises when importing solutions created in v1 or in Practice Mode.
 
-### [Website] Make side-exercise mentoring opt-in:
+## [Website] Make side-exercise mentoring opt-in:
```

Comment:
Colon at end of headings are not needed.
Same for other headings it's used on.
Not part of this change I know, but while you're editing the line...?

Reply:
Sorry, just recalled this is still in draft; I'll stop now!